pkg/rehearse/jobs.go

```diff
@@ -111,8 +111,20 @@ func makeRehearsalPresubmit(source *prowconfig.Presubmit, repo string, prNumber
 	rehearsal.Context = fmt.Sprintf("ci/rehearse/%s/%s/%s", repo, branch, shortName)
 	rehearsal.RerunCommand = defaultRehearsalRerunCommand
 
-	gitrefArg := fmt.Sprintf("--git-ref=%s@%s", repo, branch)
-	rehearsal.Spec.Containers[0].Args = append(source.Spec.Containers[0].Args, gitrefArg)
+	orgRepo := strings.Split(repo, "/")
+
+	// We will never hit different length because it will fail while we are loading the prowconfig.
```

Comment:
Alternatively, we could define:

```go
type orgRepo struct {
    org, repo string
}
```

Use this when we load and downstream from the loading code, we will have the type system help us not make any mistakes or need to make any comments like this

Reply:
we get the `org/repo` from the key of the `map[string][]Presubmit` type... I wish we had a better way to get the values but for now this is the best that we can do.. either way we have to `.Split` it in some point of the code.

Reply:
Right, I was just saying `Split()` it quickly and send the `struct` afterwords. No biggie.